File: tests/index.test.py
import pytest
from atlas_index.core.index import AtlasIndex

@pytest.mark.asyncio
async def test_discover_services():
    index = AtlasIndex(facilitator_url='https://facilitator.payai.network')
    services = await index.discover(category='AI', network='base')
    assert isinstance(services, list)

File: tests/index.test.ts
import { describe, it, expect } from 'vitest';
import { AtlasIndex } from '../core/index';

describe('AtlasIndex', () => {
  it('should create instance', () => {
    const index = new AtlasIndex({
      facilitatorUrl: 'https://facilitator.payai.network',
    });
    expect(index).toBeDefined();
  });
});


